src/pkg/path/path.go

```diff
@@ -213,6 +213,14 @@ func (pb Builder) Dir() *Builder {
 	}
 }
 
+func (pb Builder) LastElem() string {
+	if len(pb.elements) == 0 {
+		return ""
+	}
+
+	return pb.elements[len(pb.elements)-1]
```

Comment:
really makes me wish for negative indexing sometimes

Reply:
I'm a little surprised the `/x/slices` package doesn't have this (and head, tail, etc) in it.